### PR TITLE
Update Chinese translation of manual

### PR DESCRIPTION
**Pull request by yusancky**

### CODE REVIEW

File: translation/zh-CN/manual-zh-CN/src/languages_supported.md [hunk 1]
```diff
@@ -43,13 +48,13 @@
 | Zig             | [maxxnino/tree-sitter-zig](https://github.com/maxxnino/tree-sitter-zig)                                                                                        |
 
 ## 结构化文本格式
-
-| 语言 | 使用的解析器                                                             |
+           1
```
Comment (yusancky) [inline translation/zh-CN/manual-zh-CN/src/languages_supported.md:51]:
```suggestion
```

File: translation/zh-CN/manual-zh-CN/src/usage.md [hunk 2]
```diff
@@ -1,36 +1,54 @@
...
 
-Difftastic会递归地浏览这两个文件，对同名的文件进行差异分析。
+Difftastic 会递归地浏览这两个文件夹，对同名的文件进行差异分析。
+
+对比的文件夹间有许多未改变的文件时，`--skip-unchanged`选项会十分有用。
+
+### 从 stdin 读取
 
-当对比的文件夹之间许多未改变的文件时，`--skip-unchanged`选项将会十分有用。
+您可以通过指定 `-` 作为文件路径从 stdin 读取文件。
```
Comment (yusancky) [inline translation/zh-CN/manual-zh-CN/src/usage.md:23]:
```suggestion
您可以通过指定 `-` 作为文件路径从标准输入（stdin）读取文件。
```

**Comment by yusancky** [comment]
I'm sorry! For my own reasons, I have only updated the translation of `languages_supported.md` and the previous documentation. 

As for the latter, I don't have the time and energy to finish it recently. Of course, if I have time in the future, I will continue to be actively involved in updating.

**Comment by yusancky** [comment]
Additionally, I suggest that when updating `languages_supported.md` in the future, you can update the translation table at the same time.

**Comment by Wilfred** [comment]
Thanks for the PR! Any improvements are always welcome :)

**Pull request merged**